src/test/regress/expected/aux_ao_rels_stat.out

```diff
@@ -0,0 +1,219 @@
+CREATE TABLE ao_table (id int,
+ fname text,
+ lname text,
+ address1 text,
+ address2 text,
+ city text,
+ state text,
+ zip text)
+WITH (appendonly=true, orientation=column)
+DISTRIBUTED BY (id);
+CREATE INDEX ON ao_table (id);
+-- Bunch insert into AOCS table that affects auxiliary relations
+INSERT INTO ao_table (id, fname, lname, address1, address2, city, state, zip)
+SELECT i, 'Jon_' || i, 'Roberts_' || i, i || ' Main Street', 'Apartment ' || i, 'New York', 'NY', i::text
+FROM generate_series(1, 100) AS i;
+-- Wait for current backend to send locally accumulated statistics to collector
+SELECT pg_sleep(1);
+ pg_sleep 
+----------
+ 
+(1 row)
+
+-- Wait for stats collector to update accumulated statistics
+DO $$
+DECLARE
+    start_time timestamptz := clock_timestamp();
+    updated bool;
+BEGIN
+    FOR i IN 1 .. 300 LOOP
+        SELECT (sum(seq_scan) > 0) INTO updated
+        FROM gp_dist_random('pg_stat_user_tables') pgsut
+        JOIN pg_appendonly pgao ON pgsut.relid = pgao.segrelid
```

Comment:
Do we have guarantees that `segrelid` is equal among all segments? If not, we should also wrap `pg_appendonly` with `gpdist_random`.

Reply:
AFAICS relation oid are the same for the same tables on master and segments